gradle/wrapper/gradle-wrapper.properties

```
@@ -1,6 +1,6 @@
 #Sat Jun 12 13:49:20 IRDT 2021
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
```

We can upgrade it to 7.3.1.
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.1-bin.zip
```